apps/astarte_housekeeping/lib/astarte_housekeeping/queries.ex

```diff
@@ -1109,6 +1126,8 @@ defmodule Astarte.Housekeeping.Queries do
   end
 
   def set_datastream_maximum_storage_retention(realm_name, new_retention) do
+    realm_name = CQLUtils.realm_name_to_keyspace_name(realm_name, Config.astarte_instance_id!())
```

Comment:
For all that's not covered from the comment above on `Database.connect`, I'd like to see the correct naming for the same reason mentioned above.
If you do the substitution inline inside the query it's fine, but if you assign it to a variable then the variable (and all its uses below) should be named `keyspace`, not `realm_name`. This is wrong only in some instances, please check and fix where it is.